tests/bearlib/languages/LanguageMembersTypeTests.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+
+from coalib.bearlib.languages.Language import Language
+from tests.TestUtilities import LANGUAGE_NAMES
+
+
+class LanguageDefinitionTest(unittest.TestCase):
```

Comment:
Change this into `class LanguageAttributeTypeTests`, and put this class into `tests/bearlib/languages/LanguageTest.py`